includes/data-stores/class-wc-customer-data-store.php

```diff
@@ -218,6 +218,7 @@ private function update_user_meta( $customer ) {
 			'paying_customer' => 'is_paying_customer',
 			'first_name'      => 'first_name',
 			'last_name'       => 'last_name',
+			'display_name'    => 'display_name',
```

Comment:
display_name is not meta. Why is it added here?

Reply:
Not sure about the meta here before, I thought it was part of customer data. Thanks for reminding me.